packages/core/src/constants/chains.ts

```diff
@@ -359,7 +385,7 @@ export const defaultChains: Chain[] = [mainnet, ropsten, rinkeby, goerli, kovan]
 
 export const defaultL2Chains: Chain[] = [
   arbitrum,
-  arbitrumRinkeby,
+  arbitrumGoerli,
```

Comment:
Should we keep both Arbitrum Rinkeby and Goerli?

Reply:
Yeah that's a good question - the arbitrumRinkeby chain will be deprecated soon, but support is still rolling out from service providers to move to arbitrumGoerli. So maybe change this to include both until deprecation?

Reply:
Now that Infura/Alchemy provide RPC endpoints for ArbGoerli, I'd think it would make sense to encourage new developers to switch over to the new Goerli default